Maintenance/test_handling/create_testresult_page

```diff
@@ -524,6 +525,7 @@ EOF
 	    ($platform_is_optimized{$pf}) = ($tmp[1] =~ m|([-/]x?O[1-9])|);
 	    $_ = <PLATFORM_INFO>;
 	    chomp;
+	    my $build_type = $platform_is_optimized{$pf} ? "RELEASE" : "DEBUG";
```

Comment:
`RELEASE` vs. `DEBUG` is not enough to easily distinguish between the two, visually:

![Screenshot_20240911_154526](https://github.com/user-attachments/assets/99bd6be2-0f18-4000-a98f-6fe472c5fe85)

<details><summary>And by the way, can you spot the one issue in those results? (click!)</summary>

![Screenshot_20240911_154526](https://github.com/user-attachments/assets/c6a2698b-cf1a-4dd4-919c-ee6115c2d11e)

</details> 

Maybe different colors? Or icons?